app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
+
+<p>You can confirm your account email through the link below:</p>
+
```

Comment:
tab this p tag in please

Reply:
This file is installed by default not by us, we will ask a coach about what to do with this file